content/docs/source-git/_index.md

```diff
@@ -58,6 +60,16 @@ any way you want:
 * Script it yourself
 
 
+### How packit treats `git commit`s
```

Comment:
Hm, it's not clear why this topic is important. Could you explain more?

```suggestion
### How does Packit handle git commits?
```

Reply:
Maintainers need to know how patch files are going to be created out of the git commits and what packit exactly does to their spec file. Also, people who try source-git ask me this all the time.